bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/SharedDataClientWrapper.java

```diff
@@ -52,6 +52,11 @@ public BulkMutationWrapper createBulkMutation(String tableId) {
     return delegate.createBulkMutation(tableId);
```

Comment:
return createBulkMutation(tableId, 0);

Reply:
I think it's cleaner when all the code goes through a single entry point when creating the object. https://stackoverflow.com/questions/581873/best-way-to-handle-multiple-constructors-in-java

Reply:
IIUC this stackoverflow question is saying different ctors should all call the same ctor? It didn't say we should only have 1 ctor.

Reply:
I guess it doesn't matter too much